src/Symfony/Bridge/PhpUnit/SymfonyTestsListenerWithReturnTypes.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit;
+
+use PHPUnit\Framework\Test;
+use PHPUnit\Framework\TestListener;
+use PHPUnit\Framework\TestListenerDefaultImplementation;
+use PHPUnit\Framework\TestSuite;
+use PHPUnit\Framework\Warning;
+
+/**
+ * Collects and replays skipped tests.
+ *
+ * (Version of SymfonyTestsListener for PHPUnit 7+, and PHP 7.1+.)
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @final
+ */
+class SymfonyTestsListenerWithReturnTypes implements TestListener
```

Comment:
shouldn't this be marked internal as people should only interact with SymfonyTestsListener?

Reply:
yes! see #26139